spring-restdocs-core/src/main/java/org/springframework/restdocs/cli/HttpieRequestSnippet.java

```diff
@@ -46,6 +47,8 @@
  */
 public class HttpieRequestSnippet extends TemplatedSnippet {
 
+	private CommandFormatter commandFormatter = ConcatenatingCommandFormatter.newlineSeparator();
+
 	/**
 	 * Creates a new {@code HttpieRequestSnippet} with no additional attributes.
 	 */
```

Comment:
This constructor should be deprecated and a new constructor that takes a CommandFormatter introduced